scripts/upgrade

```diff
@@ -90,6 +95,13 @@ then
     YNH_PHP_VERSION="$phpversion"
 fi
 
+# Add the config error code
+if [ -n "$(ynh_app_setting_get --app=$app --key=codeerror)" ];
+then
+    echo add the error config
+    ynh_add_config --template="nginx-code-error.conf" --destination="$nginx_extra_conf_dir/error.conf"
+fi
```

Comment:
Hmmm it should probably do something similar in `scrips/config` ? Otherwise changing the setting will have no effect until next upgrade